treeherder/perf/models.py

```diff
@@ -42,6 +42,9 @@ class PerformanceSignature(models.Model):
     option_collection = models.ForeignKey(OptionCollection, on_delete=models.CASCADE)
     suite = models.CharField(max_length=80)
     test = models.CharField(max_length=80, blank=True)
+    application = models.CharField(max_length=25, null=True,
```

Comment:
Notice: Please keep the size in sync with [schema](https://phabricator.services.mozilla.com/D51410). I suggested that the length can be smaller.

Reply:
Done.